packages/ceramic-cli/src/ceramic-logger-plugins.ts

```diff
@@ -83,4 +88,23 @@ export class LogToFiles {
             }
         }
     }
+
+    /**
+     * Returns true if it has been `MINUTES_TO_EXPIRATION` minutes since the file was created
+     * @param filePath Full path of file
+     */
+    private static _isExpired (filePath: string): boolean {
+        const { birthtime } = fs.statSync(filePath)
```

Comment:
let's use promises here as well. Instead of `fs` sync functions you can import `fs` like this:

```
const fs = require('fs').promises
```

Reply:
Good idea!